references/plugins/references.go

```diff
@@ -309,20 +422,28 @@ func (ref *MarkdownReference) CreateMarkdown(ctx context.Context, caps []types.C
 			for _, property := range properties {
 				refContent += ref.prepareParameter("#"+property.Name, property.Parameters, types.HelmCategory)
 			}
+		case types.TerraformCategory:
+			// No Terraform capability is built in.
+			continue
 		default:
 			return fmt.Errorf("unsupport capability category %s", c.Category)
 		}
-
-		title := fmt.Sprintf("# %s", capNameInTitle)
-		description := fmt.Sprintf("\n\n## Description\n\n%s", c.Description)
-		specificationIntro := fmt.Sprintf("List of all configuration options for a `%s` %s.", capNameInTitle, specificationType)
-		specificationContent := ref.generateSpecification(capName)
-		specification := fmt.Sprintf("\n\n## Specification\n\n%s\n\n%s", specificationIntro, specificationContent)
+		title := fmt.Sprintf(`---
+title:  %s
+---`, capNameInTitle)
```

Comment:
why do we change the style?

Reply:
To make it consistent with the reference doc in kubevela.io https://kubevela.io/docs/developers/references/component-types/webservice.

Reply:
Will it still work in vela show --web？

Reply:
This change should be reverted, the new title style is not good if I didn't upgrade the Markdown hosting server. Thanks.
![image](https://user-images.githubusercontent.com/2805315/117521915-f2729f80-afe2-11eb-9615-8285419a1fdf.png)